advanced/Plot data extraction demo.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plot data extraction demo\n",
+    "\n",
+    "This notebook demonstrate how to extract point data from plot using plot data extraction endpoint.\n",
+    "\n",
+    "Current version only support curve plots similar to the example which has no legend."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jupyter_innotater import *\n",
+    "import matplotlib.pyplot as plt\n",
+    "from PIL import Image, ImageDraw\n",
+    "import numpy as np\n",
+    "import os\n",
+    "from cognite.client import CogniteClient\n",
+    "import json\n",
+    "import pprint\n",
+    "pp = pprint.PrettyPrinter(indent=2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a client\n",
+    "\n",
+    "To get access to your project, replace \"yourproject\" with your project name in the next cell. \n",
+    "\n",
+    "When you create the CogniteClient below, getpass will ask for your API key in an extra password field. Simply paste it in and press shift+enter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project = 'your_project'\n",
+    "api_key = os.environ.get(\"COGNITE_CONTEXT_API_KEY\")\n",
+    "client = CogniteClient(project=project,\n",
+    "                       api_key=api_key,\n",
+    "                       client_name=\"dshub\",\n",
+    "                      )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load an plot example image from google drive\n",
+    "Load an image has a plot, draw a boundingbox on the axes of the plot and copy the value (4 digit values sperated by comma) of the boundingbox to the cell below."
```

Comment:
"Load an image which includes a plot, draw a bounding box coinciding with the axes of the plot and copy the value (4 numeric values separated by commas) of the bounding box to the cell below."